### DISCUSSION TITLE
Is ursina prosible to run on android?

### DISCUSSION THREAD

**Discussion by yumoqing** [pokepetter/ursina#128] [Q&A]
Are there anyone have try to using ursina on android platform?

**Comment by kshitijvp** [comment]
Of course not

**Comment by Raphi-2Code** [comment]
There's UrsinaForMobile by PaologGithub, check out this, it can convert ursina to an apk file but entities with transparent colors are not transparent

**Comment by Raphi-2Code** [comment]
*a little update: transparency works
